src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements AreaCalculetion {

```

Comment:
The interface name 'AreaCalculetion' is a verb. According to the checklist, interface names should not be verbs. Consider renaming it to something like 'AreaCalculator' to better reflect its purpose.